cmd/manager/main.go

```diff
@@ -57,10 +66,16 @@ func main() {
 		log.Fatal(err)
 	}
 
-	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{Namespace: namespace})
+	// Create a new Cmd to provide shared dependencies and start components at the specified
+	// namespace, with metrics exposed to the provided port and host.
+	mgr, err := manager.New(cfg, manager.Options{
+		Namespace:          namespace,
+		MapperProvider:     restmapper.NewDynamicRESTMapper,
+		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+	})
 	if err != nil {
-		log.Fatal(err)
+		log.Error(err)
+		os.Exit(1)
```

Comment:
why not use `exit(err)`?

Reply:
Maybe we could. `exit(err)` stops reporting ClusterOperator status. I suppose that is the difference?

Reply:
If failing to expose metrics merits killing the operator, then `exit(err)` seems to be the right approach.

Reply:
This `err` is related to more than just exposing metrics since it's creating the manager. I'm just curious why `log.Fatal(err)` was removed in favor of the code above?

Reply:
I realize my first comment was misleading sorry

Reply:
No my mistake! I was reading the comment on the `conversation` tab and thought this pointed to this [if statement](https://github.com/awgreene/operator-marketplace/pull/6/files#diff-9e63f7b0791edcb4a53c944552b88970R105).

Reply:
Not exposing the metrics need not be related to operator not working. So `exit(err)` is not needed I suppose?

Reply:
Please keep this as `log.Fatal(err)`.